src/Sensor/Sensor.h

```diff
@@ -2,18 +2,17 @@
 #define _SENSOR_H_
 
 #include "Particle.h"
+#include "Handleable.h"
 
-class Sensor {
+class Sensor : public Handleable {
     public:
 
         /**
-         * INTERFACE
          * Initialize sensor
          * */
         virtual void begin() = 0;
```

Comment:
i don't think we need these pure virtual method declarations in Sensor now

Reply:
oh never mind I guess you do.  This ain't java folks

Reply:
Don't assume anything with C++!!